docs/man/policy/obligations/_index.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Manage obligations
+command:
+  name: obligations
+  aliases:
+    - obl
+---
+
+Obligations enable conditional access enforcement at the Policy Enforcement Point (PEP) level and allow security administrators to enforce additional restrictions beyond basic attribute-based access control (ABAC), such as requiring multi-factor authentication (MFA), enforcing watermarking, or applying time-based access expiration.
```

Comment:
Can you please create add a task in the docs ticket to add some more depth here? I think the docusaurus docs and CLI manual here will go hand in hand when we've had time to flesh out more substance about what obligations are as a feature.